src/models.rs

```diff
@@ -39,7 +41,7 @@ pub(crate) struct TableRowField {
     pub(crate) renderer: Option<IdentString>,
 
     #[darling(default)]
-    pub(crate) format: Format,
+    pub(crate) format: HashMap<syn::Ident, syn::Lit>,
```

Comment:
this has the really nice _and totaly accidental_ behaviour that naming fields that dont exist point into the macro definition, super-nice developer experience.